StabilityMatrix.Core/Models/Packages/FluxGym.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Text.RegularExpressions;
+using StabilityMatrix.Core.Attributes;
+using StabilityMatrix.Core.Helper;
+using StabilityMatrix.Core.Helper.Cache;
+using StabilityMatrix.Core.Helper.HardwareInfo;
+using StabilityMatrix.Core.Models.FileInterfaces;
+using StabilityMatrix.Core.Models.Progress;
+using StabilityMatrix.Core.Processes;
+using StabilityMatrix.Core.Python;
+using StabilityMatrix.Core.Services;
+
+namespace StabilityMatrix.Core.Models.Packages;
+
+[Singleton(typeof(BasePackage))]
+public class FluxGym(
+    IGithubApiCache githubApi,
+    ISettingsManager settingsManager,
+    IDownloadService downloadService,
+    IPrerequisiteHelper prerequisiteHelper
+)
+    : BaseGitPackage(githubApi, settingsManager, downloadService, prerequisiteHelper),
+        ISharedFolderLayoutPackage
+{
+    public override string Name => "FluxGym";
+    public override string DisplayName { get; set; } = "FluxGym";
+    public override string Author => "cocktailpeanut";
+
+    public override string Blurb => "Dead simple FLUX LoRA training UI with LOW VRAM support";
+
+    public override string LicenseType => "N/A";
+    public override string LicenseUrl => "";
+    public override string LaunchCommand => "app.py";
+
+    public override Uri PreviewImageUri =>
+        new("https://raw.githubusercontent.com/cocktailpeanut/fluxgym/main/screenshot.png");
+
+    public override List<LaunchOptionDefinition> LaunchOptions => [LaunchOptionDefinition.Extras];
+
+    public override SharedFolderMethod RecommendedSharedFolderMethod => SharedFolderMethod.Configuration;
```

Comment:
Thank you! This should also probably be updated though 😅

Reply:
my bad. done!